### PR TITLE
Update ASP.NET Core metrics

### PR DESCRIPTION
**Pull request by JamesNK**
* Change link to built in metrics to use local documentation
* Link to dashboards on grafana.com

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/log-mon/metrics/metrics.md](https://github.com/dotnet/AspNetCore.Docs/blob/66997b9c99dd472bb17780875dbc86879b448069/aspnetcore/log-mon/metrics/metrics.md) | [ASP.NET Core metrics](https://review.learn.microsoft.com/en-us/aspnet/core/log-mon/metrics/metrics?branch=pr-en-us-31141) |

### CODE REVIEW

File: aspnetcore/log-mon/metrics/metrics.md [hunk 1]
```diff
@@ -217,7 +218,7 @@ Alternatively, enter counter category such as `kestrel` in the **Expression** in
 
 1. Follow the [installation instructions](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph) to install Grafana and connect it to a Prometheus data source.
 
-1. Follow [Creating a Prometheus graph](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph). Alternatively, download a JSON file from [aspnetcore-grafana dashboards](https://github.com/JamesNK/aspnetcore-grafana/tree/main/dashboards) to configure Grafana.
+1. Follow [Creating a Prometheus graph](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph). Alternatively, download a JSON file from [.NET team dashboards @ grafana.com](https://aka.ms/dotnet/grafana-dashboards) to configure Grafana.
```
Comment (JamesNK) [inline aspnetcore/log-mon/metrics/metrics.md:221]:
```suggestion
1. Follow [Creating a Prometheus graph](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph). Alternatively, pre-built dashboards for .NET metrics are available to download at [.NET team dashboards @ grafana.com](https://aka.ms/dotnet/grafana-dashboards). Downloaded dashboard JSON can be [imported into Grafana](https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/#import-a-dashboard).
```

File: aspnetcore/log-mon/metrics/metrics.md [hunk 2]
```diff
@@ -217,7 +218,7 @@ Alternatively, enter counter category such as `kestrel` in the **Expression** in
 
 1. Follow the [installation instructions](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph) to install Grafana and connect it to a Prometheus data source.
```
Comment (Rick-Anderson) [inline aspnetcore/log-mon/metrics/metrics.md:219]:
```suggestion
* Follow the [installation instructions](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph) to install Grafana and connect it to a Prometheus data source.
```

File: aspnetcore/log-mon/metrics/metrics.md [hunk 3]
```diff
@@ -217,7 +218,7 @@ Alternatively, enter counter category such as `kestrel` in the **Expression** in
...
 
-1. Follow [Creating a Prometheus graph](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph). Alternatively, download a JSON file from [aspnetcore-grafana dashboards](https://github.com/JamesNK/aspnetcore-grafana/tree/main/dashboards) to configure Grafana.
+1. Follow [Creating a Prometheus graph](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph). Alternatively, pre-built dashboards for .NET metrics are available to download at [.NET team dashboards @ grafana.com](https://aka.ms/dotnet/grafana-dashboards). Downloaded dashboard JSON can be [imported into Grafana](https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/#import-a-dashboard).
```
Comment (Rick-Anderson) [inline aspnetcore/log-mon/metrics/metrics.md:221]:
```suggestion
* Follow [Creating a Prometheus graph](https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-graph). Alternatively, pre-built dashboards for .NET metrics are available to download at [.NET team dashboards @ grafana.com](https://aka.ms/dotnet/grafana-dashboards). Downloaded dashboard JSON can be [imported into Grafana](https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/#import-a-dashboard).
```

Comment (Rick-Anderson) [reply aspnetcore/log-mon/metrics/metrics.md:221]: Numbered instructions are harder to follow that bullets. Looks like these don't have to be done in this order. Even when they should be done in order, ppl start at the top and work down.

File: aspnetcore/log-mon/metrics/metrics.md [hunk 4]
```diff
@@ -20,7 +20,8 @@ Metrics are numerical measurements reported over time. They're typically used to
 
 These metrics can be reported to a monitoring system at regular intervals. Dashboards can be setup to view metrics and alerts created to notify people of problems. If the web service is intended to respond to requests within 400 ms and starts responding in 600 ms, the monitoring system can notify the operations staff that the app response is slower than normal.
 
-See [ASP.NET Core metrics](https://github.com/dotnet/aspnetcore/issues/47536) for ASP.NET Core specific metrics. See [.NET metrics](/dotnet/core/diagnostics/metrics) for .NET metrics.
+> [!TIP]
```
Comment (Rick-Anderson) [inline aspnetcore/log-mon/metrics/metrics.md:23]: Not a fan of `TIP` bling but won't protest too much.

image

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**